ffi/Makefile.linux

```diff
@@ -2,7 +2,7 @@
 CXX ?= g++
 
 # -flto and --exclude-libs allow us to remove those parts of LLVM we don't use
-CXX_FLTO_FLAGS ?= -flto
+#CXX_FLTO_FLAGS ?= -flto
```

Comment:
Please ignore this for now, for some reasons I can't build llvmlite locally with lto enabled.

Reply:
IIRC from my prior experience, the reason is that older versions of GCC will ICE when linking LLVM with LTO on AArch64, and I think this can happen if you're using e.g. the system compilers on Ubuntu. Moving to a newer GCC ought to resolve this.